docs/en/manuals/font.md

```diff
@@ -167,3 +162,79 @@ For example - to generate a gradient in a shader fragment, simply write:
 `float horizontal_gradient = fract(var_texcoord0.y / texture_size_recip.w);`
 
 For more information about shader uniforms, see the [Shader manual](/manuals/shader).
+
+## Runtime generation
+
+It is possible to use runtime generation for SDF type fonts, when using TrueType (.ttf) fonts.
+This approach can greatly reduce the download size and runtime memory consumption of a Defold game.
+The small downside is a very small delay for each glyph generated at runtime.
+
+Enable the feature by setting `font.runtime_generation` in game.project.
+
+::: sidenote
+This feature is currently experimental, but with the intention to be used as the default workflow in the future.
+:::
+
+::: important
+This setting affects all .ttf fonts in the project.
+:::
+
+### Prewarming glyph cache
+
+In order to make the runtime fonts easier to use, they support prewarming of the glyph cache.
+This means the font will generate the glyphs listed in *Characters* in the font.
+
+::: sidenote
+If `All Chars` is selected, there will be no prewarming as it defeats the purpose of not having to generate load all glyphs at the same time.
+:::
+
+### Font Scripting
+
+For runtime fonts, it's possible to add or removed sub fonts.
+This is useful when a large font has been split up into multiple for different character sets (e.g. CJK)
```

Comment:
"has been split up into multiple for" maybe "has been split ip into multiple **files** for"